src/Benchmarks/JsonMiddleware.cs

```diff
@@ -53,4 +58,34 @@ public static IApplicationBuilder UseJson(this IApplicationBuilder builder)
             return builder.UseMiddleware<JsonMiddleware>();
         }
     }
+
+    public class JsonArrayPool<T> : IArrayPool<T>
```

Comment:
Why does that need to be wrapped? i.e. why not just assign `ArrayPool<char>.Shared` directly to `jsonWriter.ArrayPool`?